deployment/README.md

```diff
@@ -1,379 +1,466 @@
+
+### Configure your deployment’s databases
+
+1. [Create](https://console.cloud.google.com/datastore/) a [*Native mode*](https://cloud.google.com/datastore/docs/firestore-or-datastore) Cloud Firestore database in your `{PREFIX}-{ENV}-firebase` project (the location selected here does not need to match the region configured in your `deployment.hcl` file) 
+1. Use Terraform and CICD to create Firestore indexes, a Cloud SQL instance, user accounts and IAM role bindings
+    - Uncomment the blocks for steps 5.1 through 5.6 in [`mystudies.hcl`](/deployment/mystudies.hcl), for example:
+         ```bash
+         sed -e 's/#5# //g' -i.backup $GIT_ROOT/deployment/mystudies.hcl
+         ```
+    - Regenerate the Terraform configs and commit the changes to your repo, for example:
+         ```bash
+         tfengine --config_path=$ENGINE_CONFIG --output_path=$GIT_ROOT/deployment/terraform
+         git checkout -b database-configuration
+         git add $GIT_ROOT/deployment/terraform
+         git commit -m "Configure databases"
+         git push origin database-configuration
+         ```
+    - Once your pull request pre-submit checks have completed successfully, and you have received code review approval, merge your pull request to trigger `terraform apply`(this may take up to 20 minutes - you can view the status of the operation in the [Cloud Build history](https://console.cloud.google.com/cloud-build/builds) of your `devops` project)
+1. Configure the permissions of your SQL script bucket so that your Cloud SQL instance can import the necessary initialization scripts
+    - Uncomment the blocks for Steps 6 in [`mystudies.hcl`](/deployment/mystudies.hcl), for example:
+         ```bash
+         sed -e 's/#6# //g' -i.backup $GIT_ROOT/deployment/mystudies.hcl
+         ```
+    - Regenerate the Terraform configs and commit the changes to your repo, for example:
+         ```bash
+         tfengine --config_path=$ENGINE_CONFIG --output_path=$GIT_ROOT/deployment/terraform
+         git checkout -b sql-bucket-permissions
+         git add $GIT_ROOT/deployment/terraform
+         git commit -m "Set SQL bucket permissions"
+         git push origin sql-bucket-permissions
+         ```
+    - Once your pull request pre-submit checks have completed successfully, and you have received code review approval, merge your pull request to trigger `terraform apply`(this may take up to 10 minutes - you can view the status of the operation in the [Cloud Build history](https://console.cloud.google.com/cloud-build/builds) of your `devops` project)
+1. Initialize your MySQL databases by importing SQL scripts
+    - Upload the necessary SQL script files to the `{PREFIX}-{ENV}-mystudies-sql-import` storage bucket that you created during Terraform deployment, for example:
+         ```bash
```

Comment:
the git commands above assume that the user is within the git directory, these ones assume that the user is outside of the directory. I think it makes more sense to explicitly ask the user to navigate to their local git directory and then assume all following commands are run from that location.

Reply:
added cd $GIT_ROOT to the relevant code blocks